certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2934 @@
+            "description": "Resource not found",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Resources": {
+      "get": {
+        "tags": [
+          "Resources"
+        ],
+        "summary": "Get Resources List",
+        "description": "Gets List of all Resources",
+        "operationId": "ResourcesGetAll",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Request Body (Leave empty)",
+            "x-ms-visibility": "internal",
+            "schema": {
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Existing resources returned",
+            "schema": {
+              "$ref": "#/definitions/ResourceResponseListResponse"
+            }
+          },
+          "401": {
+            "description": "User not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Operation not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Resources"
+        ],
+        "summary": "Create a Resource",
+        "description": "Creates a Resource",
+        "operationId": "ResourcesCreate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/CreateResourceRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "New resource returned",
+            "schema": {
+              "$ref": "#/definitions/ResourceResponseResponse"
+            }
+          },
+          "400": {
+            "description": "Resource not valid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "User not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Operation not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Webhooks/{id}": {
+      "get": {
+        "tags": [
+          "Webhooks"
+        ],
+        "summary": "Get a Webhook",
+        "description": "Gets a Webhook",
+        "operationId": "Webhooks_Get",

```

Comment:
@vsorokin-ppmexpress please use [pascal casing for operationId](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) properties.